### PR TITLE
Finished up set up process. Got first picture renderred on the screen

### PR DESCRIPTION
**Pull request by Pisotski**

### CODE REVIEW

File: client/src/components/Carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Carousel(props) {
```
Comment (briansai) [inline client/src/components/Carousel.jsx]: Please use ES6 fat arraow syntax for React.

File: helpers/getPhotos.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ const updatePhotos = () => {
       console.log("Error", err);
```
Comment (briansai) [inline helpers/getPhotos.js]: Callback error so the client will know that there was an error.

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    super(props);
+    this.state = {
+        collection : ['https://s3.us-east-2.amazonaws.com/airfec/dummy-1000x1000-ArchConversation.jpg', 'https://s3.us-east-2.amazonaws.com/airfec/dummy-1000x1000-Entrance.jpg', 'https://s3.us-east-2.amazonaws.com/airfec/dummy-1000x1000-NYSkyline.jpg']
+    };
+  }
+
+  render() {
+    return (
+        <Carousel collection={this.state.collection}/>
+    )
```
Comment (justuskw) [inline client/src/components/App.jsx]: Semicolon?

File: client/src/components/Carousel.jsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+    return (
+        <img src="https://s3.us-east-2.amazonaws.com/airfec/dummy-1000x1000-NYSkyline.jpg" alt="Trulli" width="500" height="333"></img>
+    )
```
Comment (justuskw) [inline client/src/components/Carousel.jsx]: semicolin?

File: client/src/components/Carousel.jsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+  }
+
+export default Carousel;
```
Comment (justuskw) [inline client/src/components/Carousel.jsx]: No new line at end of page - style guide

File: helpers/getPhotos.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+AWS.config.update({
+  accessKeyId: config.header.accessKeyId,
+  secretAccessKey: config.header.secretAccessKey,
+  region: 'us-east-2'
+});
+
+
+const validUrlsGenerator = (urls) => {
+  urlReplacer(urls.map((img) => 'https://s3.us-east-2.amazonaws.com/airfec/' + img.Key));
```
Comment (justuskw) [inline helpers/getPhotos.js]: Can you use es6 back tics instead of concatinating?

File: helpers/getPhotos.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+  const currentUrl = (urls) => {
+    return urls[Math.floor(Math.random() * urls.length)];
+  };
+
+  for (let i = 0; i < 1001; i++) {
+    let sql = `UPDATE photos SET url='${currentUrl(arrayOfUrls)}' WHERE id='${i}'`;
+    db.dbManipulator(sql, (err, data) => {
+      if (err) {
+        console.log(err);
```
Comment (justuskw) [inline helpers/getPhotos.js]: Maybe a console.error instead of console.log

File: helpers/getPhotos.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+  };
+
+  const s3 = new AWS.S3();
+                                     
+  s3.listObjects(bucketParams, function(err, data) {
+     if (err) {
+      console.log("Error", err);
+     } else {
+      validUrlsGenerator(data.Contents);
+      // console.log(`Got ${data.Contents.length} urls from s3 server`);
```
Comment (justuskw) [inline helpers/getPhotos.js]: Delete this comment.

File: helpers/getPhotos.js [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+const updatePhotos = () => {
+
+  const bucketParams = {
+     Bucket: 'airfec'
+  };
+
+  const s3 = new AWS.S3();
+                                     
+  s3.listObjects(bucketParams, function(err, data) {
```
Comment (justuskw) [inline helpers/getPhotos.js]: Stick to es6 arrow functions.

File: helpers/getPhotos.js [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+     if (err) {
+      console.log("Error", err);
+     } else {
+      validUrlsGenerator(data.Contents);
+      // console.log(`Got ${data.Contents.length} urls from s3 server`);
+     }
+  });
+}
+
+module.exports.updatePhotos = updatePhotos;
```
Comment (justuskw) [inline helpers/getPhotos.js]: New line - style guide.

File: server/connection.js [hunk 11]
```diff
@@ -0,0 +1,13 @@
+const mysql = require('mysql');
+const mysqlConfig = require('../config.js');
+
+const connection = mysql.createConnection(mysqlConfig);
+connection.connect((err, data) => err ? console.log(err) : console.log('Connected to photoList'))
+
+const dbManipulator = (sql, callback) => {
+  connection.query(sql, (err, data) =>
+  err ? callback(err, null) : callback(null, data)
```
Comment (justuskw) [inline server/connection.js]: I like this formatting. I'm going to start doing it. Looks good.

File: server/connection.js [hunk 12]
```diff
@@ -0,0 +1,13 @@
+const mysql = require('mysql');
+const mysqlConfig = require('../config.js');
```
Comment (justuskw) [inline server/connection.js]: resolve: {
    extensions: ['.js', '.jsx']
  },

If you add this to your webpack config file after the rules part, you can take out the .js and .jsx part when importing files.

File: server/index.js [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+  db.dbManipulator(sql, (err, data) => 
+    err ? res.status(500).send() : res.status(200).send(data)
+  )
+});
+
+// suppose to update the database with pictures from s3
+// helper.updatePhotos();
+
+var PORT = PORT || 3001;
+app.listen(PORT, () => console.log(`Photos module listening on port ${PORT}!`));
```
Comment (justuskw) [inline server/index.js]: New line

File: server/index.js [hunk 14]
```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const app = express();
```
Comment (justuskw) [inline server/index.js]: Put this below the files you are importing.

File: client/src/components/App.jsx (hunk 3 above)
Comment (Pisotski) [reply client/src/components/App.jsx]: semicollon could be used without parentheses.

**Pull request merged**